### ISSUE TITLE
Date Input Calendar - Reduce motion behavior

### ISSUE THREAD

**Issue by CillianHPE** [grommet/grommet#7560]
**What does this request relate to?** 
Discovered when testing A11y WCAG Rule 2.3.3 on the Date Inut with Calendar.
User should have the ability to reduce animation behavior.

**Describe the feature you'd like to request.**
Improve the out of the box behavior in Grommet by listening for the prefers-reduced-motion rule in CSS.
